### PR TITLE
Streamline main readme, and update website versioning

### PR DESCRIPTION
**Pull request by rikoe**
This change updates and simplifies the main repository README for FDC3. This serves two purposes:
- Lowers the barrier to entry for new visitors to the repository
- Succinctly explain the purpose of FDC3 to people who arrive via NPM (#261)

As part of this PR, I have simplified the initial description of FDC3, with an emphasis on the four specifications that form part of the standard. I then show a short usage example, followed by a link to the FDC3 website for more info.

I have shortened and simplified the roadmap, and tried to bring it in line with our current goals. (It previously took up a lot of unnecessary space in the README.)

I have also taken out any documentation links that led to hardcoded version URLs (e.g. version "1.0" or "1.1") - links will now take you to whatever the latest published version is.

Example (see the entire updated file [in my fork](https://github.com/rikoe/FDC3/blob/feat/streamline-main-readme/README.md)):

image

Full list of changes:
- add clickable FDC3 logo at top of README that goes to fdc3.finos.org
- simplify description of FDC3 and link to the four specifications that make up the standard
- add installation and usage sections to the README that concisely demonstrate how to use FDC3
- remove hardcoded version URLs
- improve formatting of "Getting Involved" section to make it more readable
- simplify the roadmap, removing things already achieved, and summarising to 5 steps
- move the Contributing section to the end
- website: remove previous workaround to "hard code" docusaurus urls to the latest version, meaning we can use unversioned links in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,186 @@
...
-*Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
+Another great way to interact with the community, is to attend the quarterly [FDC3 General Meeting]: you will receive an invite by signing up to the mailing list, or you can find the meeting in the [FINOS Project Meetings Calendar].
 
-## Roadmap
-With Productization on track to land at end of Q1, the expectation is that Q2 will be focused on implementations and feedback from those implementations driving further standards.  
-### Target Items for Q2
-#### API
-* Expand the broadcast API or introduce a new channels API to support the context setting use case. 
-* Publish npm package with FDC3 API definitions.
+[fdc3@finos.org]: mailto:fdc3+subscribe@finos.org
```
Comment (mattjamieson) [inline README.md:120]: should be `mailto:fdc3@finos.org` ?

File: README.md [hunk 2]
```diff
@@ -1,105 +1,186 @@
...
-* Define the next set of FDC3 Context Data types in conjunction with/based on Financial Objects working group work.
-* Work with Financial Objects and other FINOS programs to reach consensus on how to make standardized taxonomies available for consumption, both for viewing/editing and programmatically e.g. for validation. 
-* Create comprehensive examples of use cases that show how to use the various FDC3 standards together to enable interoperability.
-* App Directory Identity and authentication
+*Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org).*
 
-## License
+# License
 
 Copyright 2017 FDC3
```
Comment (mattjamieson) [inline README.md:185]: > Copyright 2017-2021 FINOS

?

File: README.md [hunk 3]
```diff
@@ -1,105 +1,186 @@
...
+[fdc3-pmc+subscribe@finos.org]: mailto:fdc3-pmc+subscribe@finos.org
+
+# Contributing
+
+If you'd like to contribute code to this repository:
+
+1. Fork it (<https://github.com/finos/FDC3/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Read our [contribution guidelines](CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
+4. Commit your changes (`git commit -am 'add some fooBar'`)
```
Comment (mattjamieson) [inline README.md:175]: 😁

**Review by mattjamieson — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,105 +1,186 @@
...
+npm install @finos/fdc3
+
+# yarn
+yarn add @finos/fdc3
+
+# pnpm
+pnpm install @finos/fdc3
+```
+
+Note that the web application still needs to run in the context of an FDC3 desktop agent to work. For more details, please see the [relevant documentation](https://fdc3.finos.org/docs/installation).
```
[the remark this answers is not in the document]
Comment (rikoe) [reply README.md:78]: @mattjamieson that link still has to be added! There is nothing about installing the npm package right now...

File: README.md (hunk 2 above)
Comment (rikoe) [reply README.md:185]: Yeah, not sure, need FINOS guidance on this one.

Comment (mindthegab) [reply README.md:185]: @mattjamieson that is OK. I'll send a PR a later on if I have a different proposal.

**Pull request merged**